pyodide_build/config.py

```diff
@@ -213,8 +212,9 @@ def to_env(self) -> dict[str, str]:
     "cxxflags": "$(CXXFLAGS_BASE)",
     "ldflags": "$(LDFLAGS_BASE) -s SIDE_MODULE=1",
     # Rust-specific configuration
-    "pyo3_cross_lib_dir": "$(CPYTHONINSTALL)/lib",
+    "pyo3_cross_lib_dir": "$(CPYTHONINSTALL)/sysconfigdata",  # FIXME: pyodide xbuildenv stores sysconfigdata here
```

Comment:
What about `$(CPYTHONINSTALL)/lib/python3.12/`?

Reply:
We don't have that directory in our xbuildenv now. Normally sysconfigdata would be placed in that directory but we don't vendor `lib/python3.X.Y/...` part in the xbuildenv and only store sysconfigdata in the top-level directory.

```
xbuildenv/pyodide-root$ ls -R | grep ":$" | sed -e 's/:$//' -e 's/[^-][^\/]*\//--/g' -e 's/^/ /' -e 's/-/|/'
 .
 |-cpython
 |---installs
 |-----python-3.12.1 (this is CPYTHONINSTALL)
 |-------include
 |---------python3.12
 |-----------cpython
 |-----------internal
 |-------sysconfigdata
 |---------__pycache__
```


Maybe we can change the path in the xbuildenv, but for now the sysconfigdata directory works like a directory where first-party libraries are stored.